internal/pkg/template/input/type.go

```diff
@@ -62,6 +63,8 @@ func (t Type) EmptyValue() interface{} {
 		return false
 	case TypeStringArray:
 		return []interface{}{}
+	case TypeObject:
+		return make(map[string]string)
```

Comment:
@webrouse Není mi úplně jasný co to má být vlastně za typ. `map` stringů by mohla stačit? Má s tím teda problém jsonnet a hodilo by se mi asi trochu navést kde hledat problém když tu ještě jsi. 😃

![CleanShot 2022-06-16 at 14 11 41](https://user-images.githubusercontent.com/215660/174067284-d610396a-12d8-4b29-aaca-ed4690cfd73b.jpg)

Reply:
@JakubMatejka JSON bez schema je by default vzdy `map[string]interface{}` , <<<< takze toto.
(v `Go 1.18`, je uz novy alias `any = interface{}`, ale este som to nemigroval, `map[string]any`)

Reply:
Tj, z definicie JSON, je kluc vzdy string, a hodnota moze mat rozny typ.
A ked ten type nazyvame (JSON) `object`, tak by to malo zodovedat.
Do buducna budu mozno aj ine `kind = x, type=object`, napr. workspace provisioning, ....

Reply:
Inak ten objekt vyzera takto:
```
{
  "oauth_api": {
    "id": "858486005",
     "version": 3
  }
}
```

![image](https://user-images.githubusercontent.com/19371734/174068403-56de5dcc-55f5-409f-ba56-9e1424f1a405.png)

Reply:
V sablone autor pouzije
```
// config.jsonnet
{
  authorization: Input("google-drive-oauth")
}
```

UI tam posle ten objekt, CLI tam vygeneruje prazdny `{}` a vypise to na konci.

Reply:
Jo jo, to je mi jasný jak vypadá. Ok, díky.